tests/conftest.py

```diff
@@ -500,6 +518,16 @@ def cherrytrack_mock_run_info_status():
     return HTTPStatus.OK
 
 
+@pytest.fixture
```

Comment:
I find these confusing and seem to add unnecessary complexity to the test fixtures of a test. Maybe I'm missing something but generic HTTP status code as fixtures instead?

Reply:
The reason of this is so we can redefine it with a mark at the beginning of the test. That way we can fail a baracoda message if we want by redefining baracoda_mock_status to be 500, fore example.
It is split in services instead of a single generic one to be able to have control on the value returned by each individual call, so if we want to fail only baracoda, but success in cherrytrack, etc.

Reply:
Hmmm it still doesn't feel right to be redefining a mock, can you create a mark when you need it instead? `baracoda_mock_status` is returning specific too so it might be worth calling it that - `baracoda_mock_status_OK`?

Reply:
Called it like baracoda_mock_status  because it has a 'default' value for baracoda status., so it can be redefined. 
If we call it baracoda_mock_status_OK we would be redefining the OK value which it would be confusing in the test.